main/newscorm/lp_final_item.php

```diff
@@ -0,0 +1,126 @@
+                $skillList = '';
+
+                if ($userSkills) {
+                    $skill = new Skill();
+                    foreach ($userSkills as $userSkill) {
+                        $oneSkill = $skill->get($userSkill['skill_id']);
+                        $skillList .= "
+                            <div class='row'>
+                                <div class='col-md-2 col-xs-6'>
+                                    <div class='thumbnail'>
+                                      <img class='skill-badge-img' src='".$oneSkill['web_icon_path']."' >
+                                    </div>
+                                </div>
+                                <div class='col-md-8 col-xs-6'>
+                                    <h5><b>".$oneSkill['name']."</b></h5>
+                                    ".$oneSkill['description']."
+                                </div>
+                                <div class='col-md-2 col-xs-12'>
+                                    <h5><b>".get_lang('ShareWithYourFriends')."</b></h5>
+                                    <a href='http://www.facebook.com/sharer.php?u=".api_get_path(WEB_PATH)."badge/".$oneSkill['id']."/user/".$userId."' target='_new'>
+                                        <em class='fa fa-facebook-square fa-3x text-info' aria-hidden='true'></em>
+                                    </a>
+                                    <a href='https://twitter.com/home?status=".api_get_path(WEB_PATH)."badge/".$oneSkill['id']."/user/".$userId."' target='_new'>
+                                        <em class='fa fa-twitter-square fa-3x text-light' aria-hidden='true'></em>
+                                    </a>
+                                </div>
+                            </div>
+                        ";
+                    }
+                    $badgeLink .= "
+                        <div class='panel panel-default'>
+                            <div class='panel-body'>
+                                <h3 class='text-center'>".get_lang('AdditionallyYouHaveObtainedTheFollowingSkills')."</h3>
+                                $skillList
+                            </div>
+                        </div>
+                    ";
+                }
+            }
+
+            $currentScore = Category::getCurrentScore($userId, $categoryId, $courseCode, $sessionId, true);
+            Category::registerCurrentScore($currentScore, $userId, $categoryId);
+        }
+    }
+}
+
+$documentInfo = DocumentManager::get_document_data_by_id(
+    $id,
+    $courseCode,
+    true,
+    $sessionId
+);
+
+$finalItemTemplate = file_get_contents($documentInfo['absolute_path']);
+
+$finalItemTemplate = str_replace('((certificate))', $downloadCertificateLink, $finalItemTemplate);
+$finalItemTemplate = str_replace('((skill))', $badgeLink, $finalItemTemplate);
+
+$tpl = new Template(null, false, false);
```

Comment:
A small line of documentation explaining why "null, false, false" would be nice (but not mandatory)